misk/web/tabs/loader/src/containers/LoaderContainer.tsx

```diff
@@ -34,13 +38,18 @@ class LoaderContainer extends React.Component<ITabProps> {
 
   render() {
     const { adminTabs } = this.props.adminTabs
+    const { adminTabComponents } = this.props.adminTabComponents
     if (adminTabs) {
       const tabRouteComponents = Object.entries(adminTabs).map(([key,tab]) => this.buildTabRouteComponent(tab))
-      const tabLinks = Object.entries(adminTabs).map(([key,tab]) => <Link key={key} to={`/_admin/test/${tab.slug}`}>{tab.name}</Link>)
+      const tabLinks = Object.entries(adminTabs).map(([key,tab]) => <Link key={key} to={`/_admin/test/${tab.slug}`}>{tab.name}<br/></Link>)
+      console.log(adminTabs, adminTabComponents)
```

Comment:
intentional console.log?

Reply:
This file is still going through a lot of work, won't be present in final PR.